user-interface/src/main/java/life/qbic/datamanager/views/general/spreadsheet/Spreadsheet.java

```diff
@@ -754,32 +736,18 @@ private void markCellAsValid(Cell cell) {
       return; // only apply to invalid cells
     }
     cell.setCellStyle(defaultCellStyle);
-    cell.removeCellComment();
   }
 
   private void updateCellValidationStatus(Cell cell, ValidationResult validationResult) {
     if (validationResult.isValid()) {
       markCellAsValid(cell);
     } else {
-      markCellAsInvalid(cell, validationResult.errorMessage());
+      markCellAsInvalid(cell);
     }
   }
 
   private boolean hasCellValidationChanged(Cell cell, ValidationResult validationResult) {
-    if (isCellValid(cell) && validationResult.isValid()) {
-      return false;
-    }
-    if (isCellInvalid(cell) && validationResult.isInvalid()) {
-      Comment existingComment = cell.getCellComment();
-      assert Objects.nonNull(validationResult.errorMessage()) : "error message is never null";
-      if (Objects.isNull(existingComment)) {
-        // error message is never null
-        return true;
-      }
-      return !validationResult.errorMessage()
-          .equals(existingComment.getString().getString());
-    }
-    return true;
+    return !isCellValid(cell) || !validationResult.isValid();
```

Comment:
```suggestion
    return (isCellValid(cell) && validationResult.isValid()) || (!isCellValid(cell) && !validationResult.isValid());
```
has it changed not is it invalid.